web_api/core/models.py

```diff
@@ -209,15 +228,24 @@ class AccountMembership(BaseModel):
     model defines that membership.
     """
 
+    class AccountRole(models.TextChoices):
+        admin = "admin"
+        member = "member"
+
     account = models.ForeignKey(
         Account, on_delete=models.CASCADE, related_name="memberships"
     )
     user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="memberships")
+    role = models.CharField(
+        max_length=255,
+        choices=AccountRole.choices,
```

Comment:
check constraint for these choices

see: https://adamj.eu/tech/2020/01/22/djangos-field-choices-dont-constrain-your-data/